stn/stn.py

```diff
@@ -430,7 +430,7 @@ def timepoint_soft_constraints(self, node_id, task, type):
     def get_time(self, task_id, type='navigation', lower_bound=True):
         _time = None
         for i, data in self.nodes.data():
-            if task_id in data['data']['task_id'] and data['data']['type'] == type:
+            if task_id == data['data']['task_id'] and data['data']['type'] == type:
```

Comment:
probably not a good idea to override the python `type`, maybe rename to `task_type`